CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -385,6 +385,19 @@ protected function getTemplateUpdates() {
           ['name' => 'event_online_receipt', 'type' => 'html'],
         ],
       ],
+      [
+        'version' => '5.75.alpha1',
+        'upgrade_descriptor' => ts('Remove the Participant Role'),
+        'templates' => [
+          ['name' => 'event_online_receipt', 'type' => 'text'],
```

Comment:
We normally only update this section when people have to take action in order to avoid breakage, because it's such a pain to update message templates. People who haven't customized the template will still get the update even if it's not listed here.

Reply:
oh ok, thanks,  I removed those lines.